index.html

```diff
@@ -273,30 +273,42 @@ <h3>
             <tr>
               <th>Specification</th>
               <th>Description</th>
+              <th>Input Documents</th>
             </tr>
             <tr>
               <td>
-<a href="https://or13.github.io/lds-pgp2021/">PGP&nbsp;Cryptosuite</a>
+                  VC Data Integrity with PGP
               </td>
               <td>
 A cryptographic digital signature suite that utilizes Pretty Good Privacy [<a
 href="https://datatracker.ietf.org/doc/html/rfc4880">RFC4880</a>].
               </td>
+              <td>
+<a href="https://or13.github.io/lds-pgp2021/">PGP&nbsp;Cryptosuite</a>
+              </td>
             </tr>
             <tr>
               <td>
-<a href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+&nbsp;Cryptosuite</a>
+                  VC Data Integrity with BBS+
```

Comment:
The BBS+ cryptosuite might be employed by both Data Integrity and JWP.

I'm also very concerned that we're taking a technology that's intended to be generalized "Data Integrity" and we're going to do something horrible, like explicitly tie it to VCs. We need to avoid that and probably need to say something to that effect in the Charter. While we will focus on DIs usage for securing VCs, we shouldn't make it so that VCs are the only things DI can secure.